plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/BaseClickHouseConnectorTest.java

```diff
@@ -693,6 +694,43 @@ public void testCreateTableWithLongTableName()
         assertTrue(getQueryRunner().tableExists(getSession(), validTableName));
     }
 
+    @Override
+    public void testRenameSchemaToLongName()
+    {
+        // Override because the max length is different from CREATE TABLE case
```

Comment:
The comment feels stale. Or did you mean `different from CREATE SCHEMA case`?

Reply:
Yes, let me fix the comment.